app/forms/decidim/participatory_documents/suggestion_form.rb

```diff
@@ -3,8 +3,33 @@
 module Decidim
   module ParticipatoryDocuments
     class SuggestionForm < Decidim::Form
+      include Decidim::AttachmentAttributes
+      include Decidim::HasUploadValidations
+
+      MIN_BODY_LENGTH = 15
+      MAX_BODY_LENGTH = 150
```

Comment:
We need to make this configurable, as a `config_accessor` at least in the plugin (maybe also as a component configuration option). Pointing this here to not forget about it but probable to be done in another PR.